fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -189,16 +195,17 @@ primaryExpression
     | number                                                                              #numericLiteral
     | booleanValue                                                                        #booleanLiteral
     | string                                                                              #stringLiteral
+    | variable                                                                            #var
+    | primaryExpression COLLATE (identifier | string)                                     #collate
     | arrayType? '[' (expression (',' expression)*)? ']'                                  #arrayConstructor
     | value=primaryExpression '[' index=valueExpression ']'                               #arraySubscript
-    | operator = (MINUS | PLUS | BITNOT) valueExpression                                  #arithmeticUnary
     | subquery                                                                            #subqueryExpression
     | EXISTS '(' query ')'                                                                #exists
     | CASE valueExpression whenClause+ (ELSE elseExpression=expression)? END              #simpleCase
     | CASE whenClause+ (ELSE elseExpression=expression)? END                              #searchedCase
-    | CAST '(' expression AS type ')'                                                     #cast
-    | identifier                                                                          #columnReference
-    | qualifiedName                                                                       #columnReference
+
+    | columnReference                                                                     #columnRef
+    | primaryExpression jsonOperator                                                      #jsonPath
```

Comment:
well, the name of jsonPath and jsonOperator maybe confusing.

- jsonPath: a syntax like `$.a.b`
- jsonOperator: I don't know what does it means

How about just name it arrowExpr? Which could used in lambda or json.

Reply:
done